pkgs/top-level/all-packages.nix

```diff
@@ -18119,7 +18119,7 @@ in
   aacgain = callPackage ../applications/audio/aacgain { };
 
   abcde = callPackage ../applications/audio/abcde {
-    inherit (pythonPackages) eyeD3;
+    inherit (python3Packages) eyeD3;
```

Comment:
If eyeD3 is used only as application, then it should not be in `python-packages.nix`.

Reply:
Technically, it seems like a python module as well: https://eyed3.readthedocs.io/en/latest/modules.html

I think people may want to use it that way and expect to find it in `pythonPackages`.